packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Providers/ParameterProvider.cs

```diff
@@ -54,6 +55,7 @@ public ParameterProvider(InputParameter inputParameter)
             if (!inputParameter.IsRequired && !type.IsCollection)
             {
                 type = type.WithNullable(true);
+                DefaultValue = Snippet.Null;
```

Comment:
Should we set this value to `default` to handle the case for an optional parameter that is a struct ?